src/domain/entities/figma-team.ts

```diff
@@ -0,0 +1,17 @@
+export enum FigmaTeamStatus {
```

Comment:
can we link out to Figma documentation for these statuses and/or add code comments to explain the enum values?

Reply:
This has been reworked to no longer be coupled to the Figma webhook statuses.